presto-main/src/main/java/com/facebook/presto/sql/planner/plan/ApplyNode.java

```diff
@@ -87,6 +103,7 @@ public PlanNode getSubquery()
         return ImmutableList.<Symbol>builder()
                 .addAll(input.getOutputSymbols())
                 .addAll(subquery.getOutputSymbols())
+                .addAll(subqueryPredicates.keySet())
```

Comment:
I think you should not add `subquery.getOutputSymbols()` when you add subqeryPredicates